cmd/aws-sigv4-proxy/main.go

```diff
@@ -47,6 +47,7 @@ var (
 	regionOverride         = kingpin.Flag("region", "AWS region to sign for").String()
 	disableSSLVerification = kingpin.Flag("no-verify-ssl", "Disable peer SSL certificate validation").Bool()
 	idleConnTimeout        = kingpin.Flag("transport.idle-conn-timeout", "Idle timeout to the upstream service").Default("40s").Duration()
+	schemeOverride         = kingpin.Flag("scheme", "Protocol to proxy with").String()
```

Comment:
I think "scheme" is too generic of a name. I would suggest "upstream-url-scheme".

Reply:
OK, I will make that change